tools/osbuild-depsolve-dnf

```diff
@@ -326,7 +326,7 @@ def read_keys(paths, root_dir=None):
     for path in paths:
         url = urllib.parse.urlparse(path)
         if url.scheme == "file":
-            path = path.removeprefix("file://")
+            path = path[7:]  # remove file:// prefix
```

Comment:
(nipitck^100) maybe `path = path[len("file://"):]  # remove file:// prefix` ? but feel free to ignore it's already fine

Reply:
Let's get this merged and I'll make it nicer in the next PR where I enable tests?

Reply:
I resolved it for now since this is addressed in #1731.